static/less/pages/categories.less

```diff
@@ -0,0 +1,110 @@
+.categories {
+
+	body:not(.admin) & {
+		&:after {
+			content: "";
+			display: block;
+			clear: both;
+		}
+
+		li {
+			float: left;
+			margin: 0 20px 20px 0;
+			padding: 0;
+			width: ~"calc(25% - 20px)";
+			border: 1px solid #bebebe;
+			transition: all 0.125s linear;
+			border-bottom: 3px solid #126f9a;
+			border-radius: 2px;
+			box-shadow: 0 1px 10px 0 rgba(0, 0, 0, 0.125), inset 0 0 0 1px rgba(255, 255, 255, 0.75);
```

Comment:
Kindly add vendor prefix to the box-shadow property.

Reply:
Vendor prefix added to the box-shadow property.